mentorship ios/Views/Members/SendRequest.swift

```diff
@@ -88,13 +88,13 @@ struct SendRequest: View {
                 self.presentationMode.wrappedValue.dismiss()
             }))
                 .alert(isPresented: $membersViewModel.sendRequestResponseData.success) {
```

Comment:
We have a failing test (testSendRequestService), probably because we changed status code from 200 to 201.
This has happened because our mock handler (for testing networking code) assumes 200 code for success.

Reply:
@yugantarjain is passing this test necessary for the merge ?

Reply:
Yes. One quick fix would be to remove the assert statement for success == 200 from the test. Or you can also first try to do some research for configuring the status code of the mock handler... Totally up to you.